x-pack/plugins/ml/public/application/components/import_export_jobs/export_jobs_flyout/export_jobs_flyout.tsx

```diff
@@ -121,6 +122,13 @@ export const ExportJobsFlyout: FC<Props> = ({ isDisabled, currentTab }) => {
         await jobsExportService.exportDataframeAnalyticsJobs(selectedJobIds);
       }
 
+      mlUsageCollection.count(
+        selectedJobType === 'anomaly-detector'
+          ? 'exported_anomaly_detector_jobs'
+          : 'exported_data_frame_analytics_jobs',
```

Comment:
I think it's worth creating a dictionary of all IDs we use in the usage collection

Reply:
yes, I was thinking the same thing

Reply:
Updated in https://github.com/elastic/kibana/pull/107772/commits/b9a4019b808b11273ae53acd6df00142788ea28b